src/dev/flang/ast/AbstractFeature.java

```diff
@@ -1685,7 +1685,7 @@ public TypeKind defaultTypeKind()
   AbstractType constraint(Context context)
   {
     if (PRECONDITIONS) require
-      (state().atLeast(State.RESOLVED_TYPES),
```

Comment:
RESOLVING_TYPES is probably also okay? But does not matter much.

Reply:
I think this did not work. I am not completely happy since I do not fully understand why, but just went with this for now.